packages/data-portal-commons/src/lib/getCaseValues.ts

```diff
@@ -10,3 +11,13 @@ export function getCaseValues(propName: keyof Entity) {
         }
     };
 }
+
+export function getNormalizedOrganCaseValues() {
+    return (e: Entity) => {
+        if (e.cases) {
+            return _.uniq(e.cases.map((c) => getNormalizedOrgan(c) as string));
+        } else {
+            return [getNormalizedOrgan(e) as string];
```

Comment:
`getNormalizedOrgan` already returns a string, `as string` typecast is probably redundant.

```suggestion
            return _.uniq(e.cases.map((c) => getNormalizedOrgan(c)));
        } else {
            return [getNormalizedOrgan(e)];
```